src/Test/API/AbstractPluginActionsTest.php

```diff
@@ -63,16 +65,6 @@ public function testPostAccountSaveAPICredentialsReturnsErrorIfMissingEmail() {
         $this->assertFalse($response['success']);
     }
 
-    public function testPostAccountSaveAPICredentialsReturnsDataStoreEmailIfSuccessful() {
-        $email = "email";
-        $this->mockRequest->method('getBody')->willReturn(array(
-            'apiKey' => 'apiKey',
-            $email => $email
-        ));
-        $this->mockAPIClient->method('createAPISuccessResponse')->willReturn(array('email' => $email));
-        $this->mockAbstractPluginActions->login();
-    }
-
```

Comment:
This was the test without an `assert()`.  I copied it over from a different implementation and it didn't make sense to keep in the rewrite.